geopyspark/geotrellis/layer.py

```diff
@@ -232,6 +266,55 @@ def to_png_rdd(self, color_map):
 
         return create_python_rdd(self.pysc, result, ser)
 
+    def to_geotiff_rdd(self,
+                       storage_method=StorageMethod.STRIPED,
+                       rows_per_strip=None,
+                       tile_dimensions=(256, 256),
```

Comment:
Sorry, misread the structure here, the striped is the default. I am no longer curious : )